docker-compose.dev.yml

```diff
@@ -23,6 +23,9 @@ services:
       - "USAON_VTA_DB_SQLITE=true"
         # Enable the in-browser debugger: 
       - "FLASK_DEBUG=True"
+      # Creds for talking to Google:
```

Comment:
Let's leave a comment saying that these envvars are only temporary until #144 is resolved?